app/assets/javascripts/discourse/lib/to-markdown.js.es6

```diff
@@ -0,0 +1,211 @@
+import parseHTML from 'discourse/helpers/parse-html';
+
+String.prototype.trimLeft = function() {
+    return this.replace(/^\s+/,"");
+}
+
+const Tag = Ember.Object.extend({
+
+  decorate(text) {
+    if (this.prefix || this.suffix) {
+      return [this.prefix, text, this.suffix].join("");
+    }
+
+    return text;
+  },
+
+  innerMarkdown() {
+    let text = this.element.innerMarkdown();
+
+    if (!["li"].includes(this.element.name)) {
```

Comment:
Isn't this the same as `this.element.name !== 'li'`?

Reply:
few more tag names may added there in near future. maybe I should remove that for now.

Reply:
I would recommend adding the array support later if you do other tag names, yes. It would be a very small change to put it back for more tag names, but we may never do that!